docs/cloud/integrations/bi/power-bi.mdx

```diff
@@ -2,14 +2,41 @@
-</svg>
-}
->
-    Click for details
-</Card>
\ No newline at end of file
+After you connect Power BI, Elementary will automatically and continuously extend the lineage to the report/dashboard level.
+This will provide you end-to-end data lineage to understand your downstream dependencies, called exposures.
+
+### Create & authorize a Service Principal
+
+Elementary makes use of a service principal secret to authenticate with the Power BI API on your behalf.
+Please follow the [official guide by Microsoft](https://learn.microsoft.com/en-us/power-bi/developer/embedded/embed-service-principal) for creating and authorizing a service principal for Power BI.
```

Comment:
Should we include a short and simplified version of it here? Our users will find it hard to read this guide

Reply:
I don't know - a shorter version might be harder to follow cause it's less detailed. Simplified is a challenge - there are multiple steps there.
Maybe you can consider writing a unified PowerShell script that does all or most of the work (they're giving there snippets for some of the stages)

Reply:
ok not a blocker, let's release and improve as we go